tool/teleport/common/teleport.go

```diff
@@ -599,7 +601,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	case integrationConfEKSCmd.FullCommand():
 		err = onIntegrationConfEKSIAM(ccf.IntegrationConfEKSIAMArguments)
 	case integrationConfAWSOIDCIdPCmd.FullCommand():
-		err = onIntegrationConfAWSOIDCIdP(ccf.IntegrationConfAWSOIDCIdPArguments)
+		err = onIntegrationConfAWSOIDCIdP(ccf, ccf.IntegrationConfAWSOIDCIdPArguments)
```

Comment:
Why not just pass `ccf` as a single argument then and take `IntegrationConfAWSOIDCIdPArguments` from it inside of the function?

Reply:
Updated, thank you!